apps/emqx_bridge_sqlserver/README.md

```diff
@@ -0,0 +1,47 @@
+# EMQX SQL Server Bridge
+
+[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server) is a relational database management system (RDBMS) that is developed and owned by Microsoft.
+Microsoft SQL Server offers a wide range of features, including support for high availability and disaster recovery,
+integration with other Microsoft products and services, and advanced security and encryption options.
+It also provides tools for data warehousing, business intelligence, and analytics, making it a versatile and powerful database platform.
+
+The application is used to connect EMQX and Microsoft SQL Server.
+User can create a rule and easily ingest IoT data into Microsoft SQL Server by leveraging
+[EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html).
+
+<!---
+
+# Documentation
+
+- Refer to [Ingest data into Cassandra](https://docs.emqx.com/en/enterprise/v5.0/data-integration/data-bridge-cassa.html)
+  for how to use EMQX dashboard to ingest IoT data into Cassandra.
+- Refer to [EMQX Rules](https://docs.emqx.com/en/enterprise/v5.0/data-integration/rules.html)
+  for the EMQX rules engine introduction.
+
+--->
+
+# Documentation links
+
+For more information about Microsoft SQL Server, please see the [official site](https://learn.microsoft.com/en-us/sql/sql-server/?view=sql-server-ver16)
+
+# Configurations
+
+
+Please see [our official documentation](https://www.emqx.io/docs/en/v5.0/data-integration/data-bridge-sqlserver.html) for more detailed information.
```

Comment:
```suggestion
Please see [Ingest data into SQL Server](https://www.emqx.io/docs/en/v5.0/data-integration/data-bridge-sqlserver.html) for more detailed information.
```

And this document has not been published yet, so we should comment out these contents first to avoid invalid links.